### PR TITLE
add different informations about Mindee

### PR DESCRIPTION
**Pull request by fharper**
- add a new image size + reorg the logos section
- add descriptions of Mindee
- add important URLs for Mindee
- add a list of Mindee's features and their short descriptions

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -10,13 +14,61 @@
...
+- [275x160 PNG](mindee/Logo-275x160.png)
+
+### Logos (vectors)
+- [Logo black](mindee/logo.svg)
+- [Icon only](mindee/logo-icon.svg)
+
+
+## Mindee Features Descriptions
+
+**Extract Key Information from any Type of Documents with ease**
```
Comment (fakela) [inline resources/README.md:33]: **Extract Key Information from any Type of Documents with ease**: Document processing and information extraction leveraging computer vision and machine learning through REST API.

**Review by fakela — APPROVED** [review]

File: resources/README.md [hunk 2]
```diff
@@ -10,13 +14,61 @@
...
+Documment processing and information extraction leveraging computer vision and machine learning throught REST API
+
+**Developer-First Platform**
+We offer a REST API that can easily be called from any type of applications or programming languages, in addition to providing SDKs for Node.js and Python (more languages & frameworks are coming)
+
+**Off-the-Shelf APIs**
+We have pre-defined model and trained APIs for common cases like receipts, invoices and passports documents
 
+**API Builder**
```
[the remark this answers is not in the document]
Comment (fharper) [reply resources/README.md:42]: I don't see the difference

[the remark this answers is not in the document]
Comment (fakela) [reply resources/README.md:42]: I couldn't read the sentence I had to turn off dark mode to realize it wasn't one full sentence. Adding `:` would enable a user to pause and not read everything like a full sentence

File: resources/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fakela) [reply resources/README.md:33]: Here change throught to through

File: resources/README.md [hunk 3]
```diff
@@ -10,13 +14,61 @@
...
+- [Logo black](mindee/logo.svg)
+- [Icon only](mindee/logo-icon.svg)
+
+
+## Mindee Features Descriptions
+
+**Extract Key Information from any Type of Documents with ease**
+Documment processing and information extraction leveraging computer vision and machine learning throught REST API
+
+**Developer-First Platform**
```
[the remark this answers is not in the document]
Comment (fakela) [reply resources/README.md:36]: Here languages to language makes the sentence sounds better

File: resources/README.md [hunk 4]
```diff
@@ -10,13 +14,61 @@
...
+**No Hassle: Up and Running in Minutes**
+Free plan, no credit card required and off-the-shelf already trained APIs
+
+## Mindee Informations
+
+### Short description (178 characters - 24 words)
+Mindee is an API first platform that helps developers automate applications' workflows by standardizing the documents processing layer using computer vision and machine learning.
+
+### Long description (475 characters - 65 words)
+Mindee is an API first platform that helps developers automate applications' workflows by standardizing the documents processing layer using computer vision and machine learning. In addition to easily detects and extracts information using pre-trained data models for common documents (e.g., invoices, receipts, passports...), developers can easily build their own documents parsing API, thereby enabling any types of businesses to solve all document-based use cases quickly.
```
[the remark this answers is not in the document]
Comment (fakela) [reply resources/README.md]: In the second and third part of the sentence:

In addition to easily detecting and extracting information using pre-trained data models for common documents.

Last part
Change businesses to business

File: resources/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (fakela) [reply resources/README.md:42]: Okay so from your reply on slack the bold parts needs to be turned to headers.

File: resources/README.md [hunk 5]
```diff
@@ -10,13 +14,61 @@
...
+- [Icon only](mindee/logo-icon.svg)
+
+
+## Mindee Features Descriptions
+
+**Extract Key Information from any Type of Documents with ease**
+Document processing and information extraction leveraging computer vision and machine learning through REST API.
+
+**Developer-First Platform**
+We offer a REST API that can easily be called from any type of application or programming language, in addition to providing SDKs for Node.js and Python (more languages & frameworks are coming)
```
Comment (fakela) [inline resources/README.md:37]: Here languages to language makes the sentence sounds better

**Pull request merged**